### PR TITLE
feat: enhance issue-manager skill with project guidance and test strategy

### PR DESCRIPTION
**Pull request by key**
## Summary
- Add project-specific package structure references to the Codebase Investigation step
- Add a Test Strategy section to the issue template covering test seams, corner cases, error scenarios, and silent failures
- Replace generic "Tests are added" DoD item with specific test coverage and silent failure checks

## Test plan
- [x] Verify SKILL.md renders correctly in GitHub
- [x] Confirm template sections flow logically: Summary → Background → Implementation → Impact → Test Strategy → DoD
- [x] Create a test issue using the skill to validate the new sections appear

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **Documentation**
  * Introduced comprehensive GitHub Issue management guidance with standardized templates and workflow documentation. Includes structured sections for issue summaries, background and motivation, implementation approaches, impact scope analysis, test strategy definition, acceptance criteria checklists, and instructions for creating and updating issues to improve consistency and quality.

### CODE REVIEW

File: .claude/skills/issue-manager/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  and silent failure scenarios (nil dependencies, empty output, fallback behavior).
+  Use table-driven tests with `t.Run()` subtests following the project convention.
+- **DoD**: Checklist format. Always include tests and CI passing. Add review criteria as needed
+
+### 4. Creating / Updating Issues
+
+#### New Issue
+
+```bash
+gh issue create --title "<title>" --body "<body>"
```
[the remark this answers is not in the document]
Comment (key) [reply .claude/skills/issue-manager/SKILL.md]: not prefer `--body-file` style.
